src/flow.ts

```diff
@@ -20,7 +20,7 @@ export type FlowOptions = {
   after?: afterFunction
 }
 
-export type Flowed = (request: RequestLike, ...extra: any[]) => Promise<any>
+export type Flowed = (request: IRequest, ...extra: any[]) => Promise<any>
 export type FlowedAndFetch = Flowed & { fetch: Flowed }
```

Comment:
This is a real weird type. It's a fetch function that owns a ... fetch function?